package.json

```diff
@@ -6,6 +6,7 @@
     "devDependencies": {
         "@changesets/cli": "2.19.0",
         "@types/jest": "27.4.0",
+        "@types/node": "17.0.8",
```

Comment:
Should opt for the lowest supported version of node (e.g. 12) and only LTS versions

Reply:
Fixed in https://github.com/SAP/open-ux-tools/pull/257/commits/c2348bd8d23c4b47495add3e57d3881c8eb5241f